<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-09QGL8M" name="com.steerlean.scenarios.store.StorePost" tests="17" failures="2" timestamp="13 Dec 2020 08:24:16 GMT" time="25.281" errors="0">
  <testcase name="TC_01" time="1.274" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_07" time="1.518" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_02" time="1.302" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_10" time="1.828" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_03" time="1.320" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_09" time="2.005" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_11" time="1.279" classname="com.steerlean.scenarios.store.StorePost">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at com.steerlean.scenarios.store.StorePost.Error(StorePost.java:312)
at com.steerlean.scenarios.store.StorePost.TC_11(StorePost.java:157)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_11 -->
  <testcase name="TC_08" time="1.792" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_06" time="1.981" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_17" time="1.318" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_16" time="1.320" classname="com.steerlean.scenarios.store.StorePost">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at com.steerlean.scenarios.store.StorePost.Error(StorePost.java:312)
at com.steerlean.scenarios.store.StorePost.TC_16(StorePost.java:217)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_16 -->
  <testcase name="TC_05" time="1.798" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_12" time="1.304" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_14" time="1.300" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_13" time="1.317" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_15" time="1.321" classname="com.steerlean.scenarios.store.StorePost"/>
  <testcase name="TC_04" time="1.304" classname="com.steerlean.scenarios.store.StorePost"/>
</testsuite> <!-- com.steerlean.scenarios.store.StorePost -->
